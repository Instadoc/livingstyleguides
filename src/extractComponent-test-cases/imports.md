---
name: Component Name
category: Component Category
---

This is a specimen with external imports:

```specimen.js
import './imports.import-1.js'
import './imports.import-2.js'
var specimen = 1
```
