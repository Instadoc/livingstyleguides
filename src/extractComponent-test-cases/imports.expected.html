<p>This is the first specimen:</p>
<pre><code class="language-specimen-1.html">&#x3C;b>Specimen 1&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-1.css">b { color: red }
</code></pre>
<p>This is a specimen with external imports:</p>
<pre><code class="language-specimen-2.html">import './imports.import-1.html'
&#x3C;b>Specimen 2&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-2.js">import './imports.import-2.js'
import './imports.import-3.js'
var specimen = 2
</code></pre>
<pre><code class="language-specimen-2.css">import './imports.import-4.css'
b { color: green }
</code></pre>